libs/service-portal/access-control/delegations/src/components/access/AccessHeader/AccessHeader.css.ts

```diff
@@ -1,37 +1,27 @@
-import { theme, themeUtils } from '@island.is/island-ui/theme'
+import { themeUtils } from '@island.is/island-ui/theme'
 import { style } from '@vanilla-extract/css'
 
-const gap = theme.spacing[6]
-
-export const container = style({
-  display: 'grid',
-  gridTemplateColumns: '1fr',
-  rowGap: gap,
-  columnGap: gap,
+export const firstColumn = style({
+  width: '100%',
+  flex: 1,
```

Comment:
~For future work, this looks like a pretty perfect use case for the (old school) Grid layout components